assets/js/components/ClusterDetails/ProviderIcon.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { EOS_HELP } from 'eos-icons-react';
+
+import AwsLogo from '@static/aws-logo.svg';
+import AzureLogo from '@static/azure-logo.svg';
+import GcpLogo from '@static/gcp-logo.svg';
+import NutanixLogo from '@static/nutanix-logo.svg';
+import KvmLogo from '@static/suse-kvm-logo.svg';
+
+const providerToLabel = {
+  aws: 'AWS',
+  azure: 'Azure',
+  gcp: 'GCP',
+  nutanix: 'Nutanix',
+  kvm: 'KVM',
+};
+
+const providerToIcon = {
+  aws: AwsLogo,
+  azure: AzureLogo,
+  gcp: GcpLogo,
+  nutanix: NutanixLogo,
+  kvm: KvmLogo,
+};
+
+const ProviderIcon = ({ provider }) => (
+  <span>
+    {providerToIcon[provider] ? (
+      <img
+        src={providerToIcon[provider]}
+        style={{ display: 'inline', height: '1em', paddingRight: '0.5em' }}
```

Comment:
style? maybe have a look on tailwind classes. Most probably there is already some class to do what you want.
But I'm not sure of that